src/fprime/cookiecutter_templates/cookiecutter-fprime-component/hooks/post_gen_project.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import print_function
+
+import datetime
+import os
+from os.path import join
+from fprime.fbuild.settings import IniSettings
+from fprime.fbuild.builder import Build
+from pathlib import Path
+import textwrap
+
+
+def replace_contents(filename, what, replacement, count=1):
+    with open(filename) as fh:
+        changelog = fh.read()
+    with open(filename, "w") as fh:
+        fh.write(changelog.replace(what, replacement, count))
+
+
+def remove_line(filename, removal):
+    with open(filename, "r") as f:
+        lines = f.readlines()
+        f.close()
+    with open(filename, "w") as f:
+        for line in lines:
+            if line != removal:
+                f.write(line)
+        f.close()
+
+
+def update_sdd():
```

Comment:
@Joshua-Anderson We typically place these in the SDD.  It is "best" to generate an aux documentation file from the dictionary that can be linked to....but that doesn't work anymore.

Reply:
@Joshua-Anderson and @LeStarch So should I just leave it as is then, or is there something I should change?

Reply:
> It is "best" to generate an aux documentation file

Are the html dictionary files not the aux documentation files?
Right now this SDD section is identical to the HTML dictionary docs so it feels a bit redundant.

Reply:
So do you think it would be best to remove this from the HTML documentation files?

Reply:
No, the html files are autogenerated from the F' dictionary, so we get them for "free". We're definitely not removing them, I'm just clarifying with Michael that they're not sufficient by themselves, since it seems like the information overlaps with the SDD command and event descriptions

Reply:
Let's keep this section in the SDD. You can't build per-component dictionaries so having a high level dictionary in the SDD is the most convenient place to reference a component's ports/commands/channels/events/etc.